README.md

```diff
@@ -177,6 +177,38 @@ docker build . -t helium-java
 
 ```
 
+### Deploy Helium Java to Azure App Service
+
+Create and configure App Service on Linux using java11
```

Comment:
why are these instructions in the java repo instead of the helium repo?

Reply:
Added them here since the deploy step uses maven to do so, which seems more specific to in helium java.